src/cloud-api-adaptor/podvm/Dockerfile.podvm.rhel

```diff
@@ -57,7 +59,7 @@ COPY . /src
 
 WORKDIR /src/cloud-api-adaptor/podvm
 
-RUN LIBC=gnu make image
+RUN LIBC=gnu ORG_ID=${ORG_ID} ACTIVATION_KEY=${ACTIVATION_KEY} make image
```

Comment:
I see, we do have it in `Dockerfile.podvm_builder.rhel` but it was missing in here. Nice...

Reply:
yep...